auditbeat/magefile.go

```diff
@@ -107,19 +107,11 @@ func Config() error {
 
 // Fields generates fields.yml and fields.go files for the Beat.
 func Fields() {
```

Comment:
@andrewkroh Is there a reason this `Fields()` method does not return an error?

Reply:
By following the pattern I was using then these tasks will be parallelized. That only works when there are no deps between them.

```
func beatFieldsGo() error {
    return mage.GenerateFieldsGo("beat")
}

func Fields() {
    mg.Deps(beatsFieldsGo, moduleFieldsGo, fieldsYML)
}
```

I was calling `fieldsYML` after `libbeatAndAuditbeatCommonFieldsGo` because both were writing to `./fields.yml` (something I fixed in my massive `mage` all the things PR). Maybe you still need to do that.

Reply:
Got it. The part I was missing is that Deps will parallise it. And as Deps does not return an error, based on the code I assume it just panics instead.